_data-prepper/pipelines/configuration/sources/selective-download.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Selective download
```

Comment:
Should we clarify this title? Something like "Selective download for S3 sources?"